apps/meteor/app/lib/server/lib/verificationCheck.ts

```diff
@@ -0,0 +1,33 @@
+import type { IRoom, IMessage } from '@rocket.chat/core-typings';
+import { LivechatRooms } from '@rocket.chat/models';
+import { RoomVerificationState, isOmnichannelRoom } from '@rocket.chat/core-typings';
+import { OmnichannelVerification } from '@rocket.chat/core-services';
+
+import { callbacks } from '../../../../lib/callbacks';
+
+callbacks.add(
+	'afterSaveMessage',
+	async function (message: IMessage, room: IRoom) {
+		if (!isOmnichannelRoom(room)) {
+			return;
+		}
+
+		if (message.u._id !== room.v._id) {
+			return;
+		}
+		if (room.verificationStatus === 'isListeningToEmail') {
```

Comment:
```suggestion
		if (room.verificationStatus === RoomVerificationState.isListeningToEmail) {
```
Let's use the enum we had for this